packages/utils/src/Handlers.ts

```diff
@@ -1,6 +1,10 @@
 import { wrapHookToRecognizeElement, Connector } from "./wrapConnectorHooks";
 
-export type HandlersMap<T extends string> = Record<T, Handler>;
+export const defineEventListener = (
```

Comment:
mmm it's a little wierd to put this util function in a separate package when it's only being used in a single fine and it's for a very specific purpose.